db/migrate/20151213185304_add_slug_to_page.rb

```diff
@@ -0,0 +1,6 @@
+class AddSlugToPage < ActiveRecord::Migration
+  def change
+    add_column :pages, :slug, :string, null: false
```

Comment:
Yeah, that's exactly what I'm trying to fix at this very moment. Didn't have the time to fix it before someone noticed it ;)